package.json

```diff
@@ -14,6 +14,13 @@
 		"cloudflare:deploy:menu": "cd ./apps/menu && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-menu",
 		"cloudflare:deploy:product": "cd ./apps/product && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-product",
 		"cloudflare:deploy:reviews": "cd ./apps/reviews && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-reviews",
+		"cloudflare:preview:cart": "npm run --prefix ./apps/cart",
```

Comment:
I think is better to remove cloudflare from cloudflare:preview:* etc. etc. because is not related to it.

Reply:
good point, done :slightly_smiling_face::+1: